apertium-dan.dan.dix

```diff
@@ -60688,6 +60688,961 @@ TODO:
+<e lm="skidengul">                     <i>skidengul</i><par n="klar__adj"/></e>
+<e lm="skifertækket">                  <i>skifertække</i><par n="øve/t__adj"/></e>
+<e lm="skiffertækket">                 <i>skiffertække</i><par n="øve/t__adj"/></e>
+<e lm="skikket">                       <i>skikke</i><par n="øve/t__adj"/></e>
+<e lm="skjoldet">                      <i>skjolde</i><par n="øve/t__adj"/></e>
+<e lm="skoldmoden">                    <i>skoldmod</i><par n="besked/en__adj"/></e>
+<e lm="skolefri">                      <i>skolefri</i><par n="klar__adj"/></e>
+<e lm="skolemoden">                    <i>skolemod</i><par n="besked/en__adj"/></e>
+<e lm="skridfri">                      <i>skridfri</i><par n="klar__adj"/></e>
+<e lm="skridsikker">                   <i>skridsik</i><par n="sik/ker__adj"/></e>
+<e lm="skrubbet">                      <i>skrubbe</i><par n="øve/t__adj"/></e>
+<e lm="skrupforelsket">                <i>skrupforelske</i><par n="øve/t__adj"/></e>
+<e lm="skrupskør">                     <i>skrupskør</i><par n="tør__adj"/></e>
+<e lm="skrækblandet">                  <i>skrækblande</i><par n="øve/t__adj"/></e>
+<e lm="skrækslagen">                   <i>skrækslag</i><par n="besked/en__adj"/></e>
+<e lm="skråstribet">                   <i>skråstribe</i><par n="øve/t__adj"/></e>
+<e lm="skudklar">                      <i>skudklar</i><par n="klar__adj"/></e>
+<e lm="skudsikker">                    <i>skudsik</i><par n="sik/ker__adj"/></e>
+<e lm="skulderbred">                   <i>skulderbred</i><par n="klar__adj"/></e>
+<e lm="skydegal">                      <i>skydegal</i><par n="klar__adj"/></e>
+<e lm="skydeglad">                     <i>skydeglad</i><par n="stolt__adj"/></e>
+<e lm="skydeklar">                     <i>skydeklar</i><par n="klar__adj"/></e>
+<e lm="skyggefuld">                    <i>skyggefuld</i><par n="klar__adj"/></e>
+<e lm="skyldbevidst">                  <i>skyldbevidst</i><par n="stolt__adj"/></e>
+<e lm="skyldsbevidst">                 <i>skyldsbevidst</i><par n="stolt__adj"/></e>
+<e lm="skæmtefuld">                    <i>skæmtefuld</i><par n="klar__adj"/></e>
+<e lm="skæppeskøn">                    <i>skæppeskøn</i><par n="grøn__adj"/></e>
+<e lm="skærbar">                       <i>skærbar</i><par n="klar__adj"/></e>
+<e lm="skævhovedet">                   <i>skævhovede</i><par n="øve/t__adj"/></e>
+<e lm="skævmundet">                    <i>skævmunde</i><par n="øve/t__adj"/></e>
+<e lm="slagen">                        <i>slag</i><par n="besked/en__adj"/></e>
+<e lm="slasket">                       <i>slaske</i><par n="øve/t__adj"/></e>
+<e lm="sleben">                        <i>sleb</i><par n="besked/en__adj"/></e>
+<e lm="slebet">                        <i>sleb</i><par n="besked/en__adj"/></e>
+<e lm="slikken">                       <i>slik</i><par n="druk/ken__adj"/></e>
+<e lm="slikket">                       <i>slikke</i><par n="øve/t__adj"/></e>
```

Comment:
lemma -et, pardef -en

Reply:
"slebet" skulle heller ikke have været med. Tak for rettelserne.